src/target/riscv/riscv.c

```diff
@@ -4653,28 +4645,28 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 	},
 	{
 		.name = "dmi_read",
-		.handler = riscv_dmi_read,
+		.handler = handle_riscv_dmi_read,
 		.mode = COMMAND_ANY,
 		.usage = "address",
 		.help = "Perform a 32-bit DMI read at address, returning the value."
```

Comment:
Optional detail:

While making changes here, you can also improve the help messages, if you like.

```suggestion
		.help = "Read and return 32-bit value from the given address on the RISC-V DMI bus."
```

Reply:
Addressed.